graphics/macgui/macfontmanager.h

```diff
@@ -25,6 +25,7 @@
 #include "common/language.h"
 #include "graphics/fonts/bdf.h"
 #include "graphics/fontman.h"
+#include "gui/widgets/richtext.h"
```

Comment:
What is the purpose of this include? Moreover. it is utterly incorrect to include gui/ in graphics/ as this adds circular dependency.

Reply:
fixed